### PR TITLE
pysyndna

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

**Comment by charles-cowart** [comment]
@antgonza https://github.com/qiita-spots/qp-woltka/pull/23/files#diff-74a4463d1da3fd6db5893bab907a6ed4e1ccabef35c642f17fbc28c1edc966a5 is an empty file. is this okay?

**Comment by antgonza** [comment]
Well, that was the output of the log, meaning there was no output so ... yes, it's fine.

File: setup.py [hunk 1]
```diff
@@ -55,6 +55,8 @@
                         'h5py >= 2.3.1', 'biom-format',
                         # forcing 0.1.4 because Qiita uses this version
                         'woltka @ https://github.com/'
-                        'qiyunzhu/woltka/archive/refs/tags/v0.1.4.zip'],
+                        'qiyunzhu/woltka/archive/refs/tags/v0.1.4.zip',
+                        'pysyndna @ https://github.com/AmandaBirmingham/'
```
Comment (charles-cowart) [inline setup.py:59]: Is it all right to leave the reference pointing to Amanda's repo? Or should pysyndna be forked into qiita-spots?

**Review by charles-cowart — CHANGES_REQUESTED** [review]
Everything looks good. Just one comment - should the reference to Amanda's repo be replaced w/a reference to a forked copy in biocore or qiita-spots?

File: setup.py (hunk 1 above)
Comment (antgonza) [reply setup.py:59]: If we fork, we will fork to biocore; however, we have no idea when or if this will happen.

**Review by charles-cowart — APPROVED** [review]

**Pull request merged**